.github/workflows/build.yml

```
@@ -26,7 +26,7 @@ jobs:
     # Run on our self-hosted machines
     runs-on: self-hosted
     container:
-      image: firedrakeproject/firedrake-env:latest
+      image: firedrakeproject/firedrake-test:latest
```

```suggestion
      image: firedrakeproject/firedrake-env:latest
```

Wait until ready to merge